sql/updates/latest-dev.sql

```diff
@@ -222,3 +221,110 @@ ALTER TABLE _timescaledb_catalog.continuous_agg_migrate_plan_step
+DROP FUNCTION IF EXISTS @extschema@.add_retention_policy(REGCLASS, "any", BOOL, INTERVAL);
+DROP FUNCTION IF EXISTS @extschema@.add_compression_policy(REGCLASS, "any", BOOL, INTERVAL);
+DROP FUNCTION IF EXISTS @extschema@.add_continuous_aggregate_policy(REGCLASS, "any", "any", INTERVAL, BOOL);
+DROP FUNCTION IF EXISTS @extschema@.add_job(REGPROC, INTERVAL, JSONB, TIMESTAMPTZ, BOOL, REGPROC);
+
+DROP VIEW IF EXISTS timescaledb_information.jobs;
+DROP VIEW IF EXISTS timescaledb_information.job_stats;
+DROP VIEW IF EXISTS timescaledb_experimental.policies;
+
+-- rebuild _timescaledb_config.bgw_job
+CREATE TABLE _timescaledb_config.bgw_job_tmp AS SELECT * FROM _timescaledb_config.bgw_job;
+ALTER EXTENSION timescaledb DROP TABLE _timescaledb_config.bgw_job;
+ALTER EXTENSION timescaledb DROP SEQUENCE _timescaledb_config.bgw_job_id_seq;
+ALTER TABLE _timescaledb_internal.bgw_job_stat DROP CONSTRAINT IF EXISTS bgw_job_stat_job_id_fkey;
+ALTER TABLE _timescaledb_internal.bgw_policy_chunk_stats DROP CONSTRAINT IF EXISTS bgw_policy_chunk_stats_job_id_fkey;
+
+CREATE TABLE _timescaledb_internal.tmp_bgw_job_seq_value AS SELECT last_value, is_called FROM _timescaledb_config.bgw_job_id_seq;
+
+DROP TABLE _timescaledb_config.bgw_job;
+
+CREATE SEQUENCE _timescaledb_config.bgw_job_id_seq MINVALUE 1000;
+SELECT pg_catalog.pg_extension_config_dump('_timescaledb_config.bgw_job_id_seq', '');
+SELECT setval('_timescaledb_config.bgw_job_id_seq', last_value, is_called) FROM _timescaledb_internal.tmp_bgw_job_seq_value;
+DROP TABLE _timescaledb_internal.tmp_bgw_job_seq_value;
+
+-- new table as we want it
+CREATE TABLE _timescaledb_config.bgw_job (
+  id INTEGER PRIMARY KEY DEFAULT nextval('_timescaledb_config.bgw_job_id_seq'),
+  application_name name NOT NULL,
+  schedule_interval interval NOT NULL,
+  max_runtime interval NOT NULL,
+  max_retries integer NOT NULL,
+  retry_period interval NOT NULL,
+  proc_schema name NOT NULL,
+  proc_name name NOT NULL,
+  owner name NOT NULL DEFAULT CURRENT_ROLE,
+  scheduled bool NOT NULL DEFAULT TRUE,
+  fixed_schedule bool not null default true,
+  initial_start timestamptz,
+  hypertable_id  integer REFERENCES _timescaledb_catalog.hypertable (id) ON DELETE CASCADE,
+  config jsonb ,
+  check_schema NAME,
+  check_name NAME 
+);
+
+ALTER SEQUENCE _timescaledb_config.bgw_job_id_seq OWNED BY _timescaledb_config.bgw_job.id;
+CREATE INDEX bgw_job_proc_hypertable_id_idx ON _timescaledb_config.bgw_job(proc_schema,proc_name,hypertable_id);
+
+INSERT INTO _timescaledb_config.bgw_job(
+    id, application_name, schedule_interval, 
+    max_runtime, max_retries, retry_period, 
+    proc_schema, proc_name, owner, scheduled, 
+    hypertable_id, config, check_schema, check_name, 
+    fixed_schedule
+)
+SELECT id, application_name, schedule_interval, max_runtime, max_retries, retry_period,
+    proc_schema, proc_name, owner, scheduled, hypertable_id, config, check_schema, check_name, FALSE
+FROM _timescaledb_config.bgw_job_tmp ORDER BY id;
```

Comment:
Shouldn't the default for system jobs (id < 1000) be TRUE for fixed_scheduled??

Reply:
I can change that if you think it best. I'm not sure if it needs to change for them all though: 
We currently have two such jobs, one is the telemetry job[1] and the other is the error retention job[2] (for the job_errors table). The telemetry job follows a different execution path, and then we set its next start in `ts_bgw_job_run_and_set_next_start` in the "drifting manner". So I don't think I need to make it `true` for the telemetry job. 
For the error retention job, I will change the default to `true` as you suggest

Reply:
Doing that we'll not be consistent between new created extension and updated via ALTER EXTENSION, because when you create the extension all system jobs (id<1000) will be marked as fixed_scheduled=true and after the update it will be fixed_scheduuled=false.